pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -151,11 +150,8 @@ def __init__(self, g_pool, source_path=None, loop=False, *args, **kwargs):
         self.timestamps = self.timestamps
 
         # set the pts rate to convert pts to frame index. We use videos with pts writte like indecies.
-        if self.buffering:
-            self.buffered_decoder = self.container.get_buffered_decoder(self.video_stream, dec_batch=50, dec_buffer_size=200)
-            self.next_frame = self.buffered_decoder.get_frame()
-        else:
-            self.next_frame = self._next_frame()
+        self.buffered_decoder = self.container.get_buffered_decoder(self.video_stream, dec_batch=50, dec_buffer_size=200)
+        self.next_frame = self.buffered_decoder.get_frame()
```

Comment:
Do we want to have buffered decoding even if we do not use the file source for playback, e.g. background circle marker detection?